extensions/grapher/src/com/google/inject/grapher/AbstractInjectorGrapher.java

```diff
@@ -81,6 +84,14 @@ public GrapherParameters setEdgeCreator(EdgeCreator edgeCreator) {
       this.edgeCreator = edgeCreator;
       return this;
     }
+
+    public SubgraphCreator getSubCreator() {
```

Comment:
and throughout the file too: getSubCreator -> getSubgraphCreator, setSubCreator -> setSubgraphCreator.